pkg/api/handlers/compat/images_push.go

```diff
@@ -196,7 +198,7 @@ loop: // break out of for/select infinite loop
 			if tag == "" {
 				tag = "latest"
 			}
-			report.Status = fmt.Sprintf("%s: digest: %s", tag, string(digestBytes))
+			report.Status = fmt.Sprintf("%s: digest: %s size: %d", tag, string(digestBytes), size)
```

Comment:
I think that `e.Artifact.Size` could be wrong since it may refer to a layer or the config. The `size` here seems to refer to the size (in bytes) of the image's manifest (see https://github.com/moby/moby/blob/master/distribution/push_v2.go#L229).

`(*libimage.Runtime).Push(....)` already returns the manifest but the podman abstractions don't pass that up.

A quicker way of getting the manifest is to probably call `(*libimage.Image).Manifest()` on the image we already looked up above (for the 404 check).

Reply:
The size is then `len(rawManifest)`.

Reply:
Want to take this over and drive it to completion.  Since I am not sure where that plugs in.

Reply:
Sure, I can do that.